### DISCUSSION TITLE
Transactions not matching actual account balance

### DISCUSSION THREAD

**Discussion by muppie** [firefly-iii/firefly-iii#7158] [General]
Hi,
Not really sure where to post so I’m trying here. Feel free to delete if it’s not suitable. 

I’m trying to manage my finances better and is trying to import my accounts, transactions, libilities into Firefly-iii. 

I’ve tried importing transactions via the importer, Nordigen API and manually adding transactions and the total still doesn’t match what is in my actual account. Sometimes it is off by 100k. I use transfers from source account to savings account etc but it still doesn’t add up. I picked the day before I started adding transactions and set that days amount in my account (say 5k on the 20th and then the imported transactions started on the 21th). 

I feel stupid and I feel like I’m missing something. Does anyone have a tip to offer that might shed light on where I should look? I’ve checked the documentation and it feels like it’s in line with how I use the the software. 

/M

**Comment by JC5** [comment]
Use the following methods to see when your balance starts to differ from what it's supposed to be:

- Make sure your opening balance is before *any* transaction in your system
- Use the "audit report" to track your running balance.
- Use the reconciliation feature to make sure all transactions are correct.

**Comment by muppie** [comment]
Thanks,

I’ll try again. 

/M

**Comment by muppie** [comment]
I figured some of it out, I think. My bank's csv contains many dates (transaction booking date, transaction date, currency date etc) and that is was calculates the values I'm comparing when I'm adding transactions. The end total still doesn't add up though so I'm still not fully sure why it doesn't match in the end.

**Comment by muppie** [comment]
It seems to be my bank’s csv file that is messed up and they will never change that. I’ll just manually add all transactions and keep an eye on the total across all accounts so everything is okay (even if I couldn’t get it to work before).

Is there a way to import based on row number and not date?

//M

**Reply by JC5** [reply]
If you use the wrong format in the date field settings (on purpose), it may revert to today's date for all transactions. That could work, perhaps?
